librz/analysis/class.c

```diff
@@ -623,7 +611,6 @@ RZ_API void rz_analysis_class_method_fini(RzAnalysisMethod *meth) {
 RZ_API void rz_analysis_class_method_recover(RzAnalysis *analysis, RzBinClass *cls, RzList /*<RzBinSymbol *>*/ *methods) {
```

Comment:
Could you please add a Doxygen comment why it substitutes certain things?

Reply:
Could you please also add `RZ_NULLABLE meth` and quick exit into `rz_analysis_class_method_fini()`?